buildconfig/Jenkins/Conda/nightly_build_and_deploy.jenkinsfile

```diff
@@ -71,6 +75,12 @@ pipeline {
                 }
             }
         }
+        stage('Update conda-recipes repository') {
+            agent { label 'master' }
```

Comment:
It'd be great to reduce the work the central node is required to do as it's a potential security issue. Now we have tht credential access in the job can we remove the restriction?

Reply:
Uhhh yes we can, I will change it to any linux node so we don't clog OSX.